src/wallet/rpc/spend.cpp

```diff
@@ -1121,10 +1124,10 @@ RPCHelpMan send()
             PartiallySignedTransaction psbtx(rawTx);
 
             // First fill transaction with our data without signing,
-            // so external signers are not asked sign more than once.
+            // so external signers are not asked to sign more than once.
             bool complete;
-            pwallet->FillPSBT(psbtx, complete, SIGHASH_DEFAULT, false, true);
-            const TransactionError err = pwallet->FillPSBT(psbtx, complete, SIGHASH_DEFAULT, true, false);
+            pwallet->FillPSBT(psbtx, complete, SIGHASH_DEFAULT, /* sign */ false, /* bip32derivs */ true);
+            const TransactionError err = pwallet->FillPSBT(psbtx, complete, SIGHASH_DEFAULT, /* sign */ true, /* bip32derivs */ false);
```

Comment:
```suggestion
            const TransactionError err = pwallet->FillPSBT(psbtx, complete, SIGHASH_DEFAULT, /*sign=*/true, /*bip32derivs=*/false);
```

& other such changes

Reply:
Fixed (after the rebase moved everything around)